src/QuantumInfo.jl

```diff
@@ -1,8 +1,13 @@
-VERSION >= v"0.4.0-dev+6521" && __precompile__()
 module QuantumInfo
-  using Cliffords
+using Cliffords
+import LinearAlgebra
```

Comment:
I would prefer to just say `using LinearAlgebra`

Reply:
I guess I would slightly prefer this too but I don't have strong feelings...  Slightly more readable without the call to LinearAlgebra every time we need it. `using LinearAlgebra` seems to be pretty quick these days...

Reply:
I'm ok with `using LinearAlgebra` here. But, I prefer `import LinearAlgebra: f1, f2`.

Some people are religious about qualifying symbols in library code. I'm not so dogmatic. But, it can really help you read the code if a function is called once or twice in a large file. E.g. At some point `Amod.b(x)` becomes uneccesary, and a new person removes it. Can he also remove `using Amod` ? If all symbols are  qualified, you can find out immediately. On the other hand, they introduce visual noise.

Two solutions are

* `import LinearAlgebra; const LA = LinearAlgebra`. (like python `import numpy as np`)
* `import LinearAlgebra: tr, ishermitian`, etc. (but how can you grep all the source for `tr` and find `map( tr, A)` and `tr(B)` without a lot of noise?)

I should add that if  symbols had been qualified I would have understood the code in `QuantumTomography` more quickly. (They weren't qualified because they were all in `Main` when the code was written.) There are many examples,  but one that stands out is `isposdef`. Some of the optimizations failed, so I began looking for alternative method for `isposdef`. After a while I realized that the method used is defined in `Convex.jl`, not in `LinearAlgebra`, and has a very different meaning. It does not take a matrix as input and return `true` or `false`.  That's why I left it qualified as `Convex.isposdef`, even though I could have imported it, or written `LinearAlgebra.isposdef`. If someone else new to `Convex` looks at it, they may not go down the wrong path like I did.

Reply:
`import LinearAlgebra: tr, ishermitian` would be my preference. Your point is well taken, though.

Reply:
I agree. But, I would use `using LinearAlgebra: tr, ishermitian` because this does not allow introducing new methods, and signals your intent to use them, but not add methods.